### PR TITLE
Add River package

### PR DESCRIPTION
**Pull request by MaxHalford**
### Description

We would like for [River](https://riverml.xyz/0.19.0/) to be available in the browser. If I understood correctly, we can contribute an entry to the packages directory for the package to be built in-tree. Please let me know if I did anything wrong!

### Checklists

- [x] Add a [CHANGELOG](https://github.com/pyodide/pyodide/blob/main/docs/project/changelog.md) entry
- [x] Add / update tests
- [x] Add new / update outdated documentation

### CODE REVIEW

**Comment by MaxHalford** [comment]
It seems the `test-core-safari` job is failing for an extraneous reason. I guess I'll rebase on main once the issue is fixed upstream.

**Comment by MaxHalford** [comment]
@hoodmane any idea why `test-packages-firefox` is failing but `test-packages-chrome` is passing? Any browser stuff I should be aware of?

**Comment by MaxHalford** [comment]
Should I add a timeout decorator? I see that's what scitkit-learn [does](https://github.com/pyodide/pyodide/blob/main/packages/scikit-learn/test_scikit-learn.py#L6-L8)

File: docs/project/changelog.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ _September 25, 2023_
 - Upgraded sourmash to 4.8.4 {pr}`4154`
 - Upgraded scikit-learn to 1.3.1 {pr}`4161`
 - Upgraded micropip to 0.5.0 {pr}`4167`
+- Added river version 0.19.0 {pr}`4197`
```
Comment (ryanking13) [inline docs/project/changelog.md:45]: Please move this to unreleased section.

**Review by ryanking13 — COMMENTED** [review]
Could you add `@pytest.mark.driver_timeout(60)` decorator? It will increase the default timeout from 20sec to 60sec.

File: docs/project/changelog.md (hunk 1 above)
Comment (MaxHalford) [reply docs/project/changelog.md:45]: My bad.

**Comment by MaxHalford** [comment]
Thanks for the tip, it's green now :). I'll let you decide if you're ok merging with or without the upstream Safari fixes.

**Comment by hoodmane** [comment]
Yes it's good to go.

**Comment by hoodmane** [comment]
Thanks @MaxHalford!

**Comment by MaxHalford** [comment]
Thanks a lot! Keep up the great work 🙏

**Pull request merged**